Signal/src/ViewControllers/AppSettingsViewController.m

```diff
@@ -108,10 +108,16 @@ - (void)dealloc
 - (void)updateTableContents
 {
     OWSTableContents *contents = [OWSTableContents new];
-    OWSTableSection *section = [OWSTableSection new];
 
     __weak AppSettingsViewController *weakSelf = self;
 
+#ifdef INTERNAL
+    OWSTableSection *internalSection = [OWSTableSection new];
+    [section addItem:[OWSTableItem softCenterLabelItemWithText:@"Internal Build"]];
```

Comment:
If you're ever confused about whether or not you have an internal build installed, you can now check the app settings view.